lib/generator.js

```diff
@@ -21,7 +21,6 @@ const {
   copyFile,
   exists,
   fetchSpec,
-  getInvalidOptions,
```

Comment:
I guess we can remove `getInvalidOptions` util?

Reply:
@derberg This is already removed :)

Reply:
but what about content of the function from `lib/utils.js` and we can remove it from mock https://github.com/asyncapi/generator/blob/master/lib/__mocks__/utils.js

Reply:
Done @derberg

Reply:
@derberg Removed Sorry , now there is not implementation of getInvalidOptions() 
![Uploading image.png…]()